java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TestJobIdGenerator.java

```diff
@@ -0,0 +1,49 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.dao.JobIdGenerator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class TestJobIdGenerator {
```

Comment:
Should we ignore the unit tests which depends on the ZooKeeper?

Reply:
When we run cicd, we will start zookeeper in each job. And the config file is /steps/zoo.cfg and clientPort is '6181'. If this clientPort change, the clientPort in this unit test should be changed.